File: Lokalano-partnerstvo/Core/Entities/Prijava.cs
using System;

namespace Core.Entities
{
    public class Prijava : BaseEntity
    {
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public DateTime DatumRodjenja { get; set; }
        public DateTime DatumPrijave { get; set; }
        public string Email { get; set; }
        // public string JMBG { get; set; }
        public string Telefon { get; set; }
        // public string Adresa { get; set; }
        public string Zanimanje { get; set; }
        public bool Pogledano { get; set; }
        public int? KursId { get; set; }  
        public Kurs Kurs { get; set; }
        public int? ObukaId { get; set; }
        public Obuka Obuka { get; set; }
        public string PrethodnoZnanje { get; set; }
        public string Objava { get; set; }
    }
}

File: Lokalano-partnerstvo/client/src/app/shared/models/prijava.ts
export interface Prijava {
  id: number;
  ime: string;
  prezime: string;
  datumRodjenja: Date;
  datumPrijave: Date;
  email: string;
  telefon: string;
  zanimanje: string;
  pogledano: boolean;
  kursId: number;
  obukaId: number;
  prethodnoZnanje: string;
  objava: string;
}

export interface PrijavaToCreate {
  ime: string;
  prezime: string;
  datumRodjenja: string;
  datumPrijave: string;
  email: string;
  telefon: string;
  zanimanje: string;
  pogledano: boolean;
  kursId: number;
  obukaId: number;
  prethodnoZnanje: string;
  objava: string;
}

export class PrijavaFormValues implements PrijavaToCreate {
  ime = '';
  prezime = '';
  datumRodjenja = Date.now.toString();
  datumPrijave = Date.now.toString();
  email = '';
  telefon = '';
  zanimanje = '';
  pogledano = false;
  kursId = null;
  obukaId = null;
  prethodnoZnanje = '';
  objava = '';

  constructor(init?: PrijavaFormValues) {
    Object.assign(this, init);
  }
}
